tests/main/views/test_marketplace.py

```diff
@@ -977,6 +977,50 @@ def test_supplier_size_visible_on_award_banner(self):
         assert 'Company size: SME' in awarded_banner.xpath('p/text()')[2]
 
 
+class TestCancelledBriefPage(BaseBriefPageTest):
+    def setup_method(self, method):
+        super(TestCancelledBriefPage, self).setup_method(method)
+        self.brief['briefs']['status'] = 'cancelled'
+
+    def test_cancelled_banner_shown_on_cancelled_brief_page(self):
```

Comment:
Might also be worth testing that the cancelled banner is not shown on not-cancelled briefs?  You might just be able to add an assertion to an existing test of non-cancelled briefs, rather than needing a whole new test.

Reply:
(Also for the other new banners - be good to test they don't show up where they shouldn't)

Reply:
This is a great idea, I will do that 👍

Reply:
For completeness, should we then also test that the award details aren't shown on non-awarded briefs?

Reply:
I made one test that checks all of that, gonna push it up in a sec :)